src/overmap.cpp

```diff
@@ -381,6 +381,11 @@ bool road_allowed(const oter_id &ter)
     return ter->has_flag( allow_override );
 }
 
+bool railroad_allowed(const oter_id &ter)
```

Comment:
Does the same as `road_allowed` above. Railroads are just different type of road, don't hardcode them as totally separate entities.